save-backend/src/main/kotlin/org/cqfn/save/backend/configs/WebSecurityConfig.kt

```diff
@@ -32,7 +32,7 @@ class WebSecurityConfig(
         // all `/internal/**` requests should be sent only from internal network
         // they are not proxied from gateway
         authorizeExchange()
-            .pathMatchers("/", "/api/projects/not-deleted", "/internal/**")
```

Comment:
@petertrr why internal endpoint is accessible outside?
So everyone can access it? Even outside?

Reply:
<img width="1405" alt="image" src="https://user-images.githubusercontent.com/58667063/148691765-62f5756b-1bd7-44c0-b7ff-902d035de46d.png">

Reply:
also please investigate this one